src/Generators/Yahoo.php

```diff
@@ -12,8 +12,8 @@ public function generate(Link $link): string
         $url = 'https://calendar.yahoo.com/?v=60&view=d&type=20';
 
         $url .= '&title='.urlencode($link->title);
-        $url .= '&st='.$link->from;
-        $url .= '&et='.$link->to;
+        $url .= '&st='.$link->from->format('Ymd\THis\Z');
```

Comment:
Hey @freekmurze and @shweshi 
`link->from` is a string (PHPDoc block in Link class is wrong). Thus tests are red:
https://travis-ci.org/spatie/calendar-links/jobs/438530152

I already fixed it in my PR https://github.com/spatie/calendar-links/pull/29 (where `Link::from` and `Link::to` are instances of `\DateTime`, as PHPDoc says)